src/server/app.js

```diff
@@ -151,6 +151,13 @@ app.use((req, res, next) => {
 	res.locals.alerts = [];
 	req.signUpDisabled = false;
 
+	if (process.env.DEPLOY_ENV === 'test' || process.env.DEPLOY_ENV === 'beta') {
+		res.locals.alerts.push({
+			level: 'danger',
```

Comment:
I think this would be a bit less harsh than a big red alert :D
```suggestion
			level: 'info',
```